tests/EpisodeManagementServiceTests/CreateUpdateEpisodeTests/CreateUpdateEpisodeTests.cs

```diff
@@ -83,11 +111,10 @@ public async Task Run_Should_Return_BadRequest_When_Episode_Is_Invalid()
             log.Log(
             LogLevel.Error,
             0,
-            It.Is<It.IsAnyType>((state, type) => state.ToString() == "Could not read episode data."),
+            It.Is<It.IsAnyType>((state, type) => state.ToString().StartsWith("Could not read episode data. Error:")),
             null,
             (Func<object, Exception, string>)It.IsAny<object>()),
             Times.Once);
-        _mockHttpRequestService.Verify(x => x.SendPost("CreateEpisodeUrl", It.IsAny<string>()), Times.Never);
```

Comment:
Don't we want to keep this assertion? If so it should be verifying no SendPost method calls are being made to any URL